common/sgx/eeid_verifier.c

```diff
@@ -313,47 +386,65 @@ static oe_result_t _eeid_verify_evidence(
         sgx_endorsements_buffer_size,
         &sgx_claims,
         &sgx_claims_length,
-        &parsed_report);
+        &parsed_report));
+
+    /* Extract various reported properties */
+    oe_report_header_t* report_header =
+        (oe_report_header_t*)sgx_evidence_buffer;
+    if (report_header->report_type != OE_REPORT_TYPE_SGX_REMOTE)
+        OE_RAISE(OE_VERIFY_FAILED);
+
+    const sgx_quote_t* sgx_quote = (const sgx_quote_t*)report_header->report;
+    const sgx_report_body_t* report_body = &sgx_quote->report_body;
+    const sgx_attributes_t* r_sgx_attributes = &report_body->attributes;
+    uint32_t r_misc_select = report_body->miscselect;
 
     const uint8_t* r_enclave_hash = parsed_report.identity.unique_id;
-    const uint8_t* r_signer_id = parsed_report.identity.signer_id;
+    const uint8_t* r_resigner_id = parsed_report.identity.signer_id;
```

Comment:
Style: what is the `r_` prefix intended to convey here? Per coding guidelines, please avoid abbreviations; consider dropping the prefix or a meaningful word e.g. `in_`?

Reply:
"reported", i.e. from the report (as opposed to computed, e.g. remeasured pages). All these names are way too long to be spelled out, especially at an 80 character line limit.

Reply:
I don't think expanding r to reported is a problem.  Line limit might result in a line break after the = but that would be ok.  "reported" is more readable.

Reply:
Done, thanks!